src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier param = new ColorSupplier();
+
+    public String getRandomBall() {

```

Comment:
```suggestion
    public Ball getRandomBall() {
```
краще повертати не строкове представлення а об'єкт, так той хто буде використовувати цей метод зможе в залежності від потреб або вивести в консоль об'єкт або зберегти його десь і тд